specs/network_details_for_hardware_inventory.adoc

```diff
@@ -0,0 +1,133 @@
+= Specification for adding network details in node hardware inventory
+
+Add network details also as a part of node hardware inventory
+
+== Problem description
+
+Hardware inventory details are used to show exact status of the particular
+node. Currently network details are missing in node hardware inventory.
+we have to add network details like IPv4, netmask, status as a part of
+node hardware inventory.
+
+== Use Cases
+
+Tendrl should show the current status of network interface.
+
+== Proposed change
+* Netifaces is used to find the network interface details.
+* Add netifaces as dependency.
+* From node_agent add a new function in pull_hardware_inventory to find network
+  interfaces details except lo, virbr0, virbr0-nic and update those details
+  into node_inventory list.
+* Create a new class in persistence module to keep network details.
+* Add a new function in persister to persist network details.
+* Modify the manager to populate the instance of newly created class using
+  node_inventory list and persist the instance in etcd.
+* Create an object in tendrl_definitions_node_agent for network details.
```

Comment:
Every 3 min once we are getting node inventory details and compare it with local inventory file, If there is any change then it will updated into etcd